docs/Developing-Wallace-(setup-guide).md

```diff
@@ -1,31 +1,41 @@
-# Getting Started: Developing Wallace
+# Developer's installation of Wallace
 
 #### Install Python 2.7
 
 You will need Python 2.7. You can check what version of Python you have by running:
     ```
     python --version
     ```
-If you do not have Python 2.7 installed, you can install it from the [Python website] (https://www.python.org/downloads/).
+If you do not have Python 2.7 installed, you can install it from the [Python website](https://www.python.org/downloads/).
 
-Or, if you know `brew`:
+Or, if you use Homebrew:
 ```
 brew install python
 ```
 
-If you have Python 3.x installed and and symlinked to the command `python` you will need to create a `virtualenv` that interprets the code as `python2.7` (because of compatibility with the `psiturk` module). Fortunately, we will be creating a virtual environment anyway, so as long as you run `brew install python` and you don't run into any errors because of your symlinks then you can just proceed with the instructions. If you do run into any errors, may god have pity on your soul (or your python installation… whichever proves to be easier). 
+If you have Python 3.*x* installed and and symlinked to the command `python`, you will need to create a `virtualenv` that interprets the code as `python2.7` (for compatibility with the `psiturk` module). Fortunately, we will be creating a virtual environment anyway, so as long as you run `brew install python` and you don't run into any errors because of your symlinks, then you can proceed with the instructions. If you do run into any errors, good luck, we're rooting for you. 
 
 #### Install Postgres using `brew`
 
+Intall postgres using homebrew:
+
 ```
 brew install postgres
 ```
 
-***
+On OS X, we recommend installing [Postgres.app](postgresapp.com) to start and stop the Postgres server.
```

Comment:
This is somewhat confusing. Homebrew is only for OSX, so it's weird to say "install it with homebrew" but then "jk, install postgres.app". I'd recommend removing the homebrew instructions if you want people to install Postgres.app.

It might also be good to include installation instructions for linux here.